src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -238,11 +240,15 @@ export class FileShareDetailComponent implements OnInit{
                         description: item.description? item.description: "--",
                         level: item.accessCapability,
                         createdAt: Utils.formatDate(item.createdAt),
-                        updatedAt: Utils.formatDate(item.updatedAt)
+                        updatedAt: Utils.formatDate(item.updatedAt),
+                        date: new Date(Utils.formatDate(item.updatedAt)).getTime()
                     }
                     this.acls.push(acl);
                 }
             })
+            this.acls.sort((previous,later)=>{
```

Comment:
ideally, this is not needed, the table component in the UI should handle this.

Reply:
Since the table component is not used and the table is being generated with plain HTML and JS the sort method will have to be used. This can be looked into when we are redesigning the application.

Reply:
LG...AB